app/src/main/java/com/github/polypoly/app/ui/game/DistanceWalkedUIComponent.kt

```diff
@@ -1,39 +1,58 @@
 package com.github.polypoly.app.ui.game
 
 import androidx.compose.foundation.background
-import androidx.compose.foundation.border
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.offset
 import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.DirectionsWalk
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment.Companion.TopCenter
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Alignment.Companion.TopEnd
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color.Companion.Black
 import androidx.compose.ui.graphics.Color.Companion.White
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.dp
-import com.github.polypoly.app.ui.game.GameActivity.Companion.formattedDistance
 import com.github.polypoly.app.ui.game.GameActivity.Companion.gameViewModel
 
 /**
  * Displays the distance walked and a button to reset it.
  */
 @Composable
 fun DistanceWalkedUIComponents() {
-    Box(modifier = Modifier.fillMaxWidth().offset(y = 80.dp)) {
-        Box(
+    fun formattedDistance(distance: Float): String {
+        return if (distance < 1000) "${"%.1f".format(distance)} m"
+        else "${"%.1f".format(distance / 1000)} km"
+    }
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .offset(y = 30.dp)
+    ) {
+        Row(
+            verticalAlignment = CenterVertically,
             modifier = Modifier
-                .fillMaxWidth()
-                .padding(16.dp)
-                .background(White)
-                .border(1.dp, Black)
-                .align(TopCenter)
+                .background(MaterialTheme.colors.primary)
+                .align(TopEnd)
+                .testTag("distanceWalkedRow")
```

Comment:
We decided with victor to put all the tags in `snake_case` in the conventions because we found that it allowed to see more quickly that it was a tag

Reply:
Fixed